druntime/test/exceptions/src/cpp_demangle.d

```diff
@@ -0,0 +1,19 @@
+import core.demangle;
+
+extern(C) char* __cxa_demangle (const char* mangled_name,
+                                                char* output_buffer,
+                                                size_t* length,
+                                                int* status) nothrow pure @trusted;
```

Comment:
This shouldn't be necessary ?

Reply:
we call `__cxa_demangle` to force link `c++` stdlib .

Reply:
Please add a comment then, and/or use a different symbol, as it might confuse the reader.